test/AngularSpec.js

```diff
@@ -240,6 +240,20 @@ describe('angular', function() {
       }
     });
 
+    it("should handle Uint8Array subarray", function() {
```

Comment:
Can you use a different type, to make the distinction between `source.length` and `source.byteLength` more obvious?